src/types/dataTypes/drupal/field_type.d.ts

```diff
@@ -23,7 +23,7 @@ export interface FieldFormattedTextWithSummary extends FieldFormattedText {
 export interface FieldLink {
   uri: string
   title: string
-  options: [string[]]
+  options: any[]
```

Comment:
What is prompting this change?

Reply:
If the JSON data returns an empty array i.e.options: [] the query tests fail when definition is options: [string[]]. It seems that the data coming back from the response is not of type [string[]].